packages/wrangler/src/api/startDevWorker/ConfigController.ts

```diff
@@ -152,7 +154,7 @@ async function resolveDevConfig(
 		testScheduled: input.dev?.testScheduled,
 		// absolute resolved path
 		persist: localPersistencePath,
-		registry: input.dev?.registry,
+		registryPath: input.dev?.registryPath,
```

Comment:
Do we need to rename this? We have other fileds in this object that are paths but aren't named with the suffix `Path` (i.e. `persist` above)

Reply:
removed the path suffix in https://github.com/cloudflare/workers-sdk/pull/10245/commits/2835b622163bde8f20285e3bb3c665a77b9c5824